wpilibc/src/main/native/include/frc/util/Color.h

```diff
@@ -761,30 +771,32 @@ class Color {
    * @return The color
    */
   static constexpr Color FromHSV(int h, int s, int v) {
-    if (s == 0) {
-      return {v / 255.0, v / 255.0, v / 255.0};
-    }
+    // loosly based on
+    // https://en.wikipedia.org/wiki/HSL_and_HSV#HSV_to_RGB
 
+    int chroma = (s * v) >> 8;
     int region = h / 30;
-    int remainder = (h - (region * 30)) * 6;
-
-    int p = (v * (255 - s)) >> 8;
-    int q = (v * (255 - ((s * remainder) >> 8))) >> 8;
-    int t = (v * (255 - ((s * (255 - remainder)) >> 8))) >> 8;
+    // remainder converted from 0-30 to roughly 0-255
```

Comment:
Put a blank line before single-line comments like this so it's clear what line of code it's referring to. Also, capitalize the first letter.